src/sentry/incidents/models.py

```diff
@@ -37,6 +39,36 @@
 from sentry.utils import metrics
 from sentry.utils.retries import TimedRetryPolicy
 
+# TODO - Move AlertsRule* and this registry to sentry.alerts
```

Comment:
Is the plan to move all of `incidents` into `sentry.alerts`?

Reply:
Sounds good - One thing I'd say is that basically everything in `incidents` is related to metric alerts. So I'm not sure that splitting the module in two makes sense, but open to it if you have some dimension you want to split on

Reply:
okay -- and nah, my split was very arbitrary. if they're all interconnected to metric alerts then let's leave them together for now 👍

Reply:
Feel free to rename the whole module though if you prefer, incidents isn't really a great name for it anymore